### PR TITLE
Updates README for publishing changes

### PR DESCRIPTION
**Pull request by oguzkocer**

### CODE REVIEW

File: README.md
Comment (oguzkocer) [inline README.md:14]: This version doesn't exist yet. I think we should publish it once we merge S3 changes to `develop`.

File: README.md
Comment (oguzkocer) [inline README.md:25]: @mokagio As far as I can tell, this is not yet implemented, is that correct? If it's not, I think we should implement it. What do you think?

File: README.md
Comment (mokagio) [inline README.md]: While we're at it...

```suggestion
* In your `build.gradle`:
```

File: README.md
Comment (mokagio) [reply README.md:25]: Correct, it is not. And agreed, there's little change people would get confused by this because of how few external users this library has, but better to be precise and only update the `README` once the code changed, too.

I can work on it if you want, although that will have to wait till the 6th or 7th of April because of Western Easter AFK days.

File: README.md
Comment (mokagio) [inline README.md]: Nitpick.
```suggestion
In the following cases, CircleCI will publish a new version with the following format to our remote Maven repo on S3:
```

or, since S3 is sort of an implementation detail:

```suggestion
In the following cases, CircleCI will publish a new version with the following format to our remote Maven repo:
```

**Review by mokagio — COMMENTED** [review]
GTG, but leaving this as only a Comment so we can wait till the CI migration work happened.

File: README.md
Comment (oguzkocer) [reply README.md:25]: @mokagio This PR is targeting the `publish-to-s3` branch, so I don't think we need to block the merge. I'll take care of the tagging issue before we merge `publish-to-s3` to `develop`.

I've addressed your suggestions, could you take another look?

Comment (oguzkocer) [reply README.md:25]: @mokagio It's been 21 days since the last comment, could you please follow up on this?

**Review by jkmassel — APPROVED** [review]

File: README.md
Comment (mokagio) [reply README.md:25]: @oguzkocer! OMG I'm so sorry I let this one slip 😢 🙇‍♂️ 🙏

**Review by mokagio — APPROVED** [review]
Super sorry for missing this. I'm not sure what happened, I must have opened this with your comment 21 days ago so GitHub marked it as read, but closed it by mistake or something.

File: README.md
Comment (oguzkocer) [reply README.md:25]: No worries :)

**Comment by mokagio** [comment]
@oguzkocer let me know if I can help you with this to atone my missed ping.

**Comment by oguzkocer** [comment]
@mokagio I am good for now, but I may ping you for more reviews soon 🤗

**Pull request merged**
